scripts/call_monitor_submission_demo.py

```diff
@@ -0,0 +1,34 @@
+"""Creating and Running Monitor Submission Workflow."""
+from firecloud import api as fapi
+from firecloud import errors as ferrors
+
+workspace_name = 'HCA_Optimus_Pipeline_MS'
+workspace_project = 'broad-firecloud-dsde'
+submission_id = '990e5930-a9b8-4f70-b220-11b7ffe702bb'
+# workflow_name has to be unique everytime
+workflow_name = 'monitor_submission_22'
+
+json_body = {
+    "namespace": 'horsefish',
+    "name": workflow_name,
+    'methodConfigVersion': 14,
+    'methodRepoMethod': {'methodName': 'monitor_submission', 'methodVersion': 14, 'methodNamespace': 'horsefish', 'methodUri': 'agora://horsefish/monitor_submission/14', 'sourceRepo': 'agora'},
+    "inputs": {"monitor_submission.submission_id": f'"{submission_id}"', "monitor_submission.terra_project": f'"{workspace_project}"', "monitor_submission.terra_workspace": f'"{workspace_name}"'},
+    "outputs": {},
+    "prerequisites": {},
+    'deleted': False,
+    'source': 'agora://horsefish/monitor_submission/14'
+}
+create_workflow_response = fapi.create_workspace_config(workspace_project, workspace_name, json_body)
```

Comment:
what does create_workspace_config do? could you add a comment here to describe the effect of this call?

Reply:
It creates a workflow, I'll add comments